cmd/event_dispatcher/main.go

```diff
@@ -23,9 +23,9 @@ func main() {
 	}
 
 	// Endpoint used by user to send data
-	serveMux.HandleFunc("/metrics", handler.Metrics)
+	serveMux.HandleFunc("/api/events", handler.Metrics)
 	// Endpoint used by clients to connect to the SSE and consume data
-	serveMux.HandleFunc("/event-dispatcher", handler.EventDispatcher)
+	serveMux.HandleFunc("/events", handler.EventDispatcher)
```

Comment:
Isn't it possible to use the same endpoint for both sending and listening to events?
The difference would be that the first one is a `POST` request, and the second a `GET` request.

Reply:
It’s possible, but it would require modifying the load balancer rules. Currently, the rule is to redirect all traffic under `/api/*` while keeping everything else hidden from public access (see [sse_broker_service.tf](https://github.com/SRGSSR/pillarbox-monitoring-infra/blob/c8cf320c868e70cc562fbe573657ddf0a1e6d0ba/pillarbox-monitoring-terraform/20-pillarbox-monitoring-app/sse_broker_service.tf#L168-L210)).

Even though your suggestion is more REST-like, I think it would tightly couple the API definition with the load balancer configuration, which would reduce flexibility. The current setup allows the API and load balancer to evolve independently.